openedx/features/colaraz_features/models.py

```diff
@@ -0,0 +1,25 @@
+
+"""
+Django models for colaraz_features app.
+"""
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+
+class ColarazUserProfile(models.Model):
+    """
+    Represents colaraz user profile.
+    """
+    class Meta:
+        app_label = 'colaraz_features'
+        verbose_name_plural = 'Colaraz user profiles'
+
+    user = models.OneToOneField(User, unique=True, db_index=True,
+                                related_name='colaraz_profile', on_delete=models.CASCADE)
+    site_identifier = models.CharField(max_length=100)
```

Comment:
`site_identifier` is the company name that will be present at the start of the domain. For example, if the domain is `pucit.courses.colaraz.com` then the site_identifier will be `pucit`.